1.data-exploration/sample_info_data_visualization.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# import the necessary packages that will be utilized
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd 
+import pathlib
+import seaborn as sns
+
+# assign the desired file a variable using pathlib.Path command
+input_file = pathlib.Path("../0.data-download/data/sample_info.csv")
+
+# set the data frame to be the desired .csv file that is read by pandas(pd) using the pd.read_csv(desired file read as a previously defined variable)
+df = pd.read_csv(input_file)
+
+# print the parameters of the read file
+print(df.shape)
+
+
+# In[2]:
+
+
+# how many samples?
+n_samples = len(df["DepMap_ID"])
+print(f"Number of Samples: {n_samples} \n")
+
+# how many genes 
+all_cancers = df["age"].unique()
+print(f"Ages sampled from: \n {all_cancers} \n")
+
+# how many different types of cancer?
+all_cancers = df["primary_disease"].unique()
+print(f"All Cancer Types: \n {all_cancers} \n")
+
+
+# In[3]:
+
+
+# "Adult" was set to 18
+# "Pediatric" was set to 0
+# "Fetus" was set to 0
```

Comment:
Is this a common metadata distinction?

I have never seen "Fetus" before, so this is a bit surprising to me

Reply:
Also, having these comments is great, but you can also code this explicitly, and then use a variable _instead of hardcoding_ later on in the script. This is more pythonic to do it this way :)

for example:

```python
adult_age = 18
ped_age = 0

# create a new data frame that will also assign the integer value of 18 to cells containing "Adult"
no_nan_age_df.loc[(df["age"] == "Adult")] = adult_age

.... and so on...
```

Reply:
I have not seen fetus before either. the assignment of the integer value of -1 to the string value of "Fetus" as it appears in the data is to create a unique integer value that has no effect on the real data and can therefor be ignored in the later function.

Reply:
gotcha! If it can be ignored, then I think we can set it to "ignored" instead of "Fetus".

Also (and I think I mentioned this later on), it'll be good to find some sort of data manifest (might be in a README or elsewhere on the DepMap download) that describes these codes. For example, -1 might actually mean missing.

If you can find where these data are codified, please add to this pull request. Having accurate metadata labels is _extremely_ important! :)